app/assess/views/filters.py

```diff
@@ -8,6 +8,7 @@ def status_to_human(status: str):
         "SUBMITTED": "Submitted",
         "COMPLETED": "Complete",
         "FLAGGED": "Flagged",
+        "STOPPED": "Stopped",
```

Comment:
Think we could replace this with the built-in Jinja filters either [capitalise](https://tedboy.github.io/jinja2/templ14.html#capitalize) or [title](https://tedboy.github.io/jinja2/templ14.html#title) just awaiting confirmation from design which is correct CC @charlesrt

Reply:
@Adam-W1 design have confirmed `title` case is the intended behaviour, so please can we remove this and use the built-in `title` filter instead?

Reply:
@Adam-W1 So I think on the basis of what design have said. `capitalize` is the correct filter to use.

Reply:
I think we still need the custom as the statuses display slightly differently on UI (including removing underscore), For example:
- "NOT_STARTED" -> "Not started"
- "COMPLETED" -> "Assessment completed".

Reply:
Could try doing that chaining a standard filter in the view e.g. `| replace("_"," ")` 

Or could replace this function with 
```
return status.capitalize().replace('_', ' ')
```
and achieve same result.

Reply:
Oh I see missed the assessment complete thing